app/templates/components/forms/form-select.hbs

```diff
@@ -0,0 +1 @@
+{{yield}}
```

Comment:
The  `no-newline` icon here but not in the file before suggests to me your editor isn’t listening to `.editorconfig`, is that something you have set up?

Reply:
Hmm 🤔 It should be set up (I use vim), but I'll double-check.